site/docs/v1/tech/messengers/generic-messenger.adoc

```diff
@@ -0,0 +1,82 @@
+---
+layout: doc
+title: Generic Messenger
+description: Adding and Configuring Messengers - Generic
+---
+:page-liquid:
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+== Set Up a Generic Messenger
+
+Currently the only supported transport for a messenger is SMS. If you need to connect to a SMS provider other than Twilio, you can use Kafka or a Generic messenger configuration.
+
+The generic messenger allows for a simple JSON REST API integration which allows you to receive a message from FusionAuth and then make an API call to any third party SMS provider you wish.
+
+image::messengers/add-messenger.png[Messenger Home Screen, width=1200]
+
+To create a new generic messenger, navigate to `icon:comments[type=fas] Messengers`. +
+Simply click `icon:plus[type=fas] Add Generic Messenger`
+
+=== Add Generic Messenger
+
+image::messengers/add-generic-messenger-home-crop.png[Add Generic Messenger Home, role=bottom-cropped, width=1200]
+
+Just complete the following fields:
+
+==== Form Fields
+
+[.api]
+[field]#Id# [optional]#Optional#::
+A unique UUID is auto generated if left blank.
+
+[field]#Name# [required]#Required#::
+For display only. Name of this generic messenger.
+
+[field]#URL# [required]#Required#::
+This is the URL of the messaging service you wish to connect to.
+
+[field]#Connect timeout# [required]#Required#::
+In milliseconds, how long to keep the socket open when connecting to the messenger
+
+[field]#Read timeout# [required]#Required#::
+In milliseconds, how long to keep the socket open when reading to the messenger.
+Must be an integer and greater than 0.
+
+[field]#Debug enabled# [optional]#Optional#::
+When enabled, each message sent using this messenger will generate a new Debug Event Log which can be viewed using the Event Log API or from the admin UI by navigating to `System > Event Log`.
```

Comment:
When giving navigation, please use [breadcrumb]#System -> Event Log# in docs.

Check all the other navigation doc in this section as well.